x-pack/plugins/elastic_assistant/server/routes/knowledge_base/constants.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const ELSER_MODEL_ID = '.elser_model_1';
```

Comment:
conisder adding a comment explaining why this should NOT be changed from `.elser_model_1`